components/brave_wallet/browser/asset_ratio_service.cc

```diff
@@ -232,6 +232,16 @@ void AssetRatioService::GetBuyUrlV1(mojom::OnRampProvider provider,
     api_request_helper_->Request("POST", sardine_token_url, payload,
                                  "application/json", true,
                                  std::move(internal_callback), request_headers);
+  } else if (provider == mojom::OnRampProvider::kTransak) {
+    GURL transak_url = GURL(kTransakURL);
+    transak_url = net::AppendQueryParameter(transak_url, "apiKey", kTransakApiKey);
```

Comment:
Yeah looks like it would be easy for someone to impersonate us with this API key. I'll need to look at what privileges are granted to this token before I can approve this.

Reply:
According to Transak, the only privileges associated with that API key are the ones related to what we're doing with it (i.e. buying crypto for a given address).

Reply:
In that case, I think this one is good to go. Thanks for checking